src/scss/components/choice-tile/_index.scss

```diff
@@ -0,0 +1,213 @@
+@import "components/box/config";
+@import "components/grid/config";
+@import "components/form/mixins/input";
+@import "./mixins/hxChoiceTile";
+@import "mixins";
+
+hx-tile {
+  @include box-size(md);
+
+  border-radius: 2px;
+  display: inline-block;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  > header {
+    font-size: 1rem;
+    font-weight: 700;
+    margin-bottom: 0;
+  }
+
+  > p {
+    font-size: 0.75rem;
+  }
+
+  &.hxSm {
+    $pad: 0.75rem;
+
+    padding: $pad;
+
+    > p {
+      display: none;
+    }
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  &.hxLg {
+    $pad: 1.75rem;
+
+    padding: $pad;
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  > header + *,
+  .hx-tile-icon + * {
+    margin-top: 0.5rem;
+  }
+
+  .hx-tile-icon {
+    hx-icon {
+      font-size: 1.75rem;
+    }
+  }
+
+  hx-icon[type="checkmark"] {
+    display: none;
+    font-size: 1rem;
+    right: 1.25rem;
+    top: 1.25rem;
+  }
+
+  &.hxSubdued {
+    @include hxChoiceTile--subdued;
+
+    &:hover {
+      @include hxChoiceTile--subdued($pseudo-state: hover);
+    }
+  }
+}
+
+.hxRow > label.hxChoice {
+  @include grid-column;
+}
+
+// Layout & Geometry
+// ========== COMMON ==========
+label.hxChoice {
+  position: relative;
+}
+
+label.hxChoice > input {
+  @include hxInput(visuallyHidden);
+
+  &:focus {
+    @include reset-focus;
+  }
```

Comment:
As it is, this results in ```label.hxChoice > input:focus:focus { ... }```, (double `:focus`) because the `reset-focus`mixin applies [its own `&:focus` selector](https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/_mixins.scss#L64) in addition to the `&:focus` here.

Please unwrap the `@include` and order reset before `hxInput()`.

```scss
label.hxChoice > input {
  @include reset-focus;
  @include hxInput(visuallyHidden);

  ...
}
```

Reply:
Changes has been done.